core/templates/pages/exploration-player-page/learner-experience/conversation-skin.component.spec.ts

```diff
@@ -2298,7 +2298,7 @@ describe('Conversation skin component', () => {
 
       expect(componentInstance.feedbackIsEnabled).toBeTrue();
       expect(componentInstance.learnerCanOnlyAttemptQuestionOnce).toBeFalse();
-      expect(componentInstance.lastInputOutputPairIsOnlyShown).toBeFalse();
+      expect(componentInstance.inputOutputHistoryIsShown).toBeFalse();
```

Comment:
Shouldn't the "sense" of inputOutputHistoryIsShown be opposite to that of lastInputOutputPairIsOnlyShown? I.e. shouldn't true <-> false?

inputOutputHistoryIsShown means we show the full conversation history.
lastInputOutputPairIsOnlyShown means we show only the last input-output pair.
They are opposites of each other. Please fix throughout.

Reply:
Updated